flake8_quotes/__init__.py

```diff
@@ -12,31 +12,53 @@ class QuoteChecker(object):
     QUOTES = {
         # When user wants only single quotes
         '\'': {
-            'good_single': '\'',
-            'good_multiline': '\'\'\'',
-            'bad_single': '"',
-            'bad_multiline': '"""',
+            'good': '\'',
+            'bad': '"',
         },
+
+        # When user wants only double quotes
         '"': {
-            'good_single': '"',
-            'good_multiline': '"""',
-            'bad_single': '\'',
-            'bad_multiline': '\'\'\'',
+            'good': '"',
+            'bad': '\'',
         },
     }
 
+    MULTILINE_QUOTES = {
+        # When user wants only single multiline quotes
+        '\'': {
+            'good': '\'\'\'',
+            'bad': '"""'
+        },
+
+        # When user wants only double multiline quotes
+        '"': {
+            'good': '"""',
+            'bad': '\'\'\''
+        }
+    }
+
+    MULTILINE_QUOTES_STRINGS = ('\'\'\'', '"""')
+
     def __init__(self, tree, filename='(none)', builtins=None):
         self.filename = filename
 
     @classmethod
     def add_options(cls, parser):
         parser.add_option('--quotes', default='\'', action='store',
                           help='Quote to expect in all files (default: \')')
-        parser.config_options.append('quotes')
+        parser.add_option('--multiline-quotes', action='store',
+                          help='Multiline quote to expect in all files (disabled by default)')
+        parser.config_options.extend(['quotes', 'multiline_quotes'])
 
     @classmethod
     def parse_options(cls, options):
+        if hasattr(options, 'multiline_quotes'):
+            multiline_quotes = options.multiline_quotes
+        else:
+            multiline_quotes = None
+
         cls.quotes = cls.QUOTES[options.quotes]
+        cls.multiline_quotes = cls.MULTILINE_QUOTES[multiline_quotes] if multiline_quotes is not None else None
```

Comment:
If you use a different error code, you can just use the quotes selected and then you don't have to worry about it being None blow (just here).